praw/models/reddit/subreddit.py

```diff
@@ -1476,6 +1486,23 @@ def update(
             "text_color": text_color,
             "text_editable": text_editable,
         }
+        if fetch:
+            _existing_data = [
+                template
+                for template in iter(self)
+                if template["id"] == template_id
+            ]
+            if len(_existing_data) != 1:
+                raise InvalidFlairTemplateID(template_id)
+            else:
+                existing_data = _existing_data[0]
+                data.update(
+                    {
+                        key: value
+                        for key, value in existing_data.items()
+                        if value is not None
+                    }
```

Comment:
I think this is the wrong way to update. This will replace all values in `data` with the corresponding values from `existing_data`.

Reply:
I will revise the method to replace the default values.